lib/src/util/__tests__/keyboardUtil.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+import { without } from 'ramda';
+
+import { rtlRender } from '../../__testutils__/rtlUtils';
+import { userEvent } from '../../__testutils__/rtlUtils';
+import { modifierKeyIsPressed, MODIFIER_KEYS } from '../keyboardUtil';
+
+describe('modifierKeyIsPressed', function () {
+  const Fixture = () => {
+    const [isPressed, setIsPressed] = useState(false);
+    const handleKeyDown = (event: React.KeyboardEvent) => {
+      event.preventDefault();
+      setIsPressed(modifierKeyIsPressed(event));
+    };
+    return <div tabIndex={1} data-testid="target" onKeyDown={handleKeyDown}>{isPressed.toString()}</div>;
+  };
+
+  const quickRender = rtlRender(Fixture, {});
+
+  it('should only return true when a modifier key is pressed', async () => {
+    const user = userEvent.setup();
+    const { getByTestId } = quickRender();
+
+    const targetEl = getByTestId('target');
+    expect(targetEl).toHaveTextContent('false');
+    targetEl.focus();
+
+    // "OS" is not playing nicely with js-dom
```

Comment:
What were the issues?  That's one of the more realistic keys so it'd be a shame to not be able to test it

Reply:
> That's one of the more realistic keys

Well, maybe.  MDN claims that means the Windows key, but at least on my Linux system that's not the case.  Maybe it is on Windows though?

Reply:
Hmm yeah it is returning false on the test 😿 
It looks like Mac uses the Meta key, but it's strange that it is still returning false when
artificially setting the type to "OS".

I'm curious, what's the difference between "Super" or "Hyper" key vs "Meta" key in Linux?

Reply:
> I'm curious, what's the difference between "Super" or "Hyper" key vs "Meta" key in Linux?

Long story short, it completely depends on your configuration.  From the sound of it the Windows key is typically mapped to Meta in the KDE desktop environment (what I use) but not in GNOME or others which are also common (where it is mapped to Super).   See https://askubuntu.com/a/19565 .

tl;dr
> Furthermore, the definition of modifier keys depends on:
> * The keyboard itself
> * The keyboard layout
> * The driver
> * The operating system's implementation

Reply:
> > I'm curious, what's the difference between "Super" or "Hyper" key vs "Meta" key in Linux?
> 
> Long story short, it completely depends on your configuration. From the sound of it the Windows key is typically mapped to Meta in the KDE desktop environment (what I use) but not in GNOME or others which are also common (where it is mapped to Super). See https://askubuntu.com/a/19565 .
> 
> tl;dr
> 
> > Furthermore, the definition of modifier keys depends on:
> > 
> > * The keyboard itself
> > * The keyboard layout
> > * The driver
> > * The operating system's implementation

Thanks!

Reply:
> As explained in that post, the key names come from a certain ridiculous keyboard from years past: ![image](https://user-images.githubusercontent.com/3630091/234291631-e484b5f1-6209-4ac4-a8f3-5ea9f9cbfc09.png)

I would love to have that keyboard just because 😆

Reply:
> I would love to have that keyboard just because

Same